src/DocumentDB.ChangeFeedProcessor.UnitTests/Logging/TraceLogProviderTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+//----------------------------------------------------------------
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.UnitTests.Logging
+{
+    using Microsoft.Azure.Documents.ChangeFeedProcessor.Logging;
+    using System.Diagnostics;
+    using System.IO;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    [Trait("Category", "Gated")]
+    public class TraceLogProviderTests
+    {
+        [Fact]
+        public void Log_WhenTraceProviderConstructedWithTraceSource()
```

Comment:
Should we update the README to include how to wire down tracing in NET6 as an example?